src/main.rs

```diff
@@ -186,13 +186,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 row.push(Cell::from(format!("{}", row_index)).style(style));
                 for column_index in minimum_column_index..=maximum_column_index {
                     let value = model
-                        .formatted_cell_value(selected_sheet as u32, row_index as i32, column_index)
+                        .get_formatted_cell_value(
+                            selected_sheet as u32,
+                            row_index as i32,
+                            column_index,
+                        )
+                        .unwrap();
```

Comment:
Add unwrap since it now returns a result

Reply:
Well we won't have unwraps in production code, but Tiron is not really production ready :)